solutions/web3-data-fetching/lib/BAYC.abi.json

```diff
@@ -0,0 +1,416 @@
+[
```

Comment:
Is this file auto generated?

Reply:
This is obtained from etherscan. It's autogenerated when you code a smart contract but when only consuming data from one it is needed to establish the connection